src/rust_connection/xauth.rs

```diff
@@ -25,11 +25,12 @@ pub(crate) enum Family {
 impl From<X11Family> for Family {
```

Comment:
Can/should this `Family` enum be replaced with a newtype around `u16`, too? It already has an `Unknown(u16)` variant, so I guess it can just be turned into such.